spec/nativeTemplateEngineBehaviors.js

```diff
@@ -50,7 +50,7 @@ describe('Native template engine', function() {
 
         it('can fetch template from <textarea> elements and data-bind on results', function () {
             var testTextAreaTemplate = ensureNodeExistsAndIsEmpty("testTextAreaTemplate", "textarea"),
-                prop = (typeof testTextAreaTemplate.innerText !== "undefined") ? "innerText" : "textContent";
+                prop = "value";
             testRenderTemplate(testTextAreaTemplate, "testTextAreaTemplate", prop);
```

Comment:
This is a possible concern. Because of the changes here, you won't be able to change a `<textarea>`-based template by changing its `textContent` after its been used. That's because the template code uses `value` to read and set the content, and once `value` is set, the browser ignores changes to `textContent`.